OpenEdXMobile/src/test/java/org/edx/mobile/view/WebViewActivityTest.java

```diff
@@ -39,24 +37,25 @@ public void test_StartWebViewDialogActivity_LoadsUrlAndShowsTitle() {
      * @param title The title to show, if any
      */
     protected static void test_StartWebViewDialogActivity_LoadsUrlAndShowsTitle(String url, String title) {
-        final WebViewDialogActivity activity =
-                Robolectric.buildActivity(WebViewDialogActivity.class)
-                        .withIntent(WebViewDialogActivity.newIntent(RuntimeEnvironment.application, url, title)).setup().get();
+        final WebViewActivity activity =
+                Robolectric.buildActivity(WebViewActivity.class)
+                        .withIntent(WebViewActivity.newIntent(RuntimeEnvironment.application, url, title)).setup().get();
         final View dialogView = Shadows.shadowOf(activity).getContentView();
         assertNotNull(dialogView);
         final WebView webView = (WebView) dialogView.findViewById(R.id.eula_webView);
         assertNotNull(webView);
         final ShadowWebView shadowWebView = Shadows.shadowOf(webView);
         assertEquals(shadowWebView.getLastLoadedUrl(), url);
-        final TextView titleView = (TextView) dialogView.findViewById(R.id.tv_dialog_title);
-        assertNotNull(titleView);
+        android.support.v7.app.ActionBar actionBar = activity.getSupportActionBar();
+        assertNotNull(actionBar);
         if (TextUtils.isEmpty(title)) {
-            Assertions.assertThat(titleView).isNotVisible();
+            assertTrue(!actionBar.isShowing());
         } else {
-            Assertions.assertThat(titleView).isVisible();
-            Assertions.assertThat(titleView).hasText(title);
+            assertTrue(actionBar.isShowing());
+            assertTrue(title.equals(activity.getTitle()));
```

Comment:
It would be better to use AssertJ Android's `assertThat` (with a static import), as we were doing before. It has specific assertion methods defined for all Android APIs.